lib/editor/components/EditorInput.js

```diff
@@ -314,7 +316,18 @@ export default class EditorInput extends Component<Props> {
                 ? <option disabled value=''>{field.required ? '-- select an option --' : '(optional)' }</option>
                 : null
               }
-              {field.options && field.options.map(o => (<option value={o.value} key={o.value}>{o.text || o.value}</option>))}
+              {field.name === 'route_type' &&
+                <optgroup label={standard}>
+                  {field.options.slice(0, 7).map(o => (
```

Comment:
Because `o` shows up so many times, I would write `.map(({ text, value }) => { ...`, and maybe also extract the `<optgroup>` rendering to its own method because of repetition.